apps/website/src/pages/docs/button.mdx

```diff
@@ -18,51 +18,92 @@ import * as AllExamples from '~/examples';
 </LiveExample>
 
-Buttons are the recipients of a user onClick action (a mouse click for browsers and a finger press on touch-sensitive devices). Buttons have states and events and should be used when a form is submitted (post) or where the button represents a clear call to action on a page. In some cases, buttons can also be used in place of href links to draw special attention to that action.
+## Appearance
+
+### Default
 
-## Button
+The default button should be used in most circumstances. If you are unsure as to what button to use, use this button!
 
 <LiveExample src='Button.default.tsx'>
   <AllExamples.ButtonDefaultExample client:load />
 </LiveExample>
 
-### Props
+### High visibility
+
+This is the button to emphasize an action applicable to an area of the page or a [dialog](dialog). High visibility buttons are used to draw the user’s attention to the main action of the page.
+There should only be one high visibility button per area of the user interface.
+
+<LiveExample src='Button.highvisibility.tsx'>
+  <AllExamples.ButtonHighVisibilityExample client:load />
+</LiveExample>
+
+### Call to action
+
+The call to action button, also referred to as the CTA, is the button that is used for most full page layouts and whole-page overlays (think Next in stepper-like interactions). This is the biggest and most noticeable button.
+One CTA button per visible page is the rule. Treat it as a large, “CLICK HERE” guide for the user.
+
+<LiveExample src='Button.cta.tsx'>
+  <AllExamples.ButtonCTAExample client:load />
+</LiveExample>
+
+### Borderless
+
+The borderless button is useful in tighter areas where using a default button would visibly cause too many boxes.
+
+The borderless button is often used by actionable icons in relation to another component.
+They are commonly used within [table](table) rows and widgets. A good example would be the close icon within a modal, toast notification, or alert.
+
+<LiveExample src='Button.borderless.tsx'>
+  <AllExamples.ButtonBorderlessExample client:load />
+</LiveExample>
+
+### Size
+
+There are 3 different sizes available, which can be applied to any button.
+
+The medium size should always be the first choice, unless there are good reasons to use the small or large version.
+
+<LiveExample src='Button.size.tsx'>
+  <AllExamples.ButtonSizeExample client:load />
+</LiveExample>
+
+## Button props
 
 <PropsTable path={frontmatter.propsPathDefault} />
 
 ## Split button
 
+Pressing this left side of the button will commit the action. Pressing the right side of the button will open a menu.
+
 <LiveExample src='SplitButton.main.tsx'>
   <AllExamples.SplitButtonMainExample client:load />
 </LiveExample>
 
-Pressing this left side of the button will commit the action. Pressing the right side of the button will open a menu. Upon selecting an option from the menu, the action immediately takes place, and the left side of the button’s label will change to match the recent selection.
-
 ### Props
```

Comment:
I started using `Properties` instead of `Props`.  @jbwriter asked how props might translate, so I figured it would be best to use the full word.

Reply:
"Props" is not just a shorthand, it's the canonical term and we should not deviate from it.
https://react.dev/learn/passing-props-to-a-component

Reply:
Ok, I will revert my branch to `Props` and will share this with Jay.